Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -71,16 +71,16 @@ WebXROpaqueFramebuffer::~WebXROpaqueFramebuffer()
     if (auto gl = m_context.graphicsContextGL()) {
 #if USE(IOSURFACE_FOR_XR_LAYER_DATA)
         if (m_opaqueTexture)
-            gl->deleteTexture(m_opaqueTexture);
+            gl->deleteTexture(m_opaqueTexture.release());
```

Comment:
I'm trying to work out if it would be better to have the m_opaqueTexture destructor call deleteTexture. But then it would need to hold on to a reference to the GLContext, which is sucky.

Reply:
I had a version that was `m_opaqueTexture.deleteObject(gl)` so exact handle didn't have to carry a pointer-sized reference to `GraphicsContextGL` around.